pandas/tests/io/test_excel.py

```diff
@@ -32,18 +32,16 @@ def frame(float_frame):
     return float_frame[:10]
 
 
-@pytest.fixture
-def frame2(float_frame):
-    float_frame = float_frame.copy()
-    float_frame.columns = ['D', 'C', 'B', 'A']
-    return float_frame[:10]
-
-
 @pytest.fixture
 def tsframe():
     return tm.makeTimeDataFrame()[:5]
 
 
+@pytest.fixture(params=[True, False])
+def merge_cells(request):
```

Comment:
This was previously a decorator for the writing tests but used in less than half of those tests, so it was adding unnecessary parametrization. By decoupling this from the class we can simply inject where necessary

Reply:
No problem - will do on next push